release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -2745,6 +2745,49 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+[id="ocp-4-15-35_{context}"]
+=== RHSA-2024:7179 - {product-title} 4.15.35 bug fix and security update
+
+Issued: 02 October 2024
+
+{product-title} release 4.15.35, which includes security updates, is now available. The list of bug fixes that are included in this update is documented in the link:https://access.redhat.com/errata/RHSA-2024:7179[RHSA-2024:7179] advisory. The RPM packages that are included in this update are provided by the link:https://access.redhat.com/errata/RHSA-2024:7182[RHSA-2024:7182] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.15.35 --pullspecs
+----
+
+[id="ocp-4-15-35-bug-fixes_{context}"]
+==== Bug fixes
+
+* Previously, when you upgraded an {product-title} cluster from 4.14 to 4.15, the `vCenterCluster` parameter was not populated with a value in the `use-connection-form.ts` configuration file. As a result, the {vmw-full} GUI did not display {vmw-full} vCenter information. With this release, an update to the Infrastructure custom resource (CR) ensures it has that the GUI checks the 'cloud-provider-config' ConfigMap for the `vCenterCluster` value. (link:https://issues.redhat.com/browse/OCPBUGS-42144[*OCPBUGS-42144*])
+
+* Previously, deploying a self-managed private hosted cluster on {aws-first} fails because the `bootstrap-kubeconfig` file uses an incorrect `kube-apiserver` port. As a result, the {aws-short} instances are provisioned cannot join the hosted cluster as nodes. With this release, the issue is fixed so this issue no longer occurs. (link:https://issues.redhat.com/browse/OCPBUGS-42214[*OCPBUGS-42214*])
```

Comment:
```suggestion
* Previously, deploying a self-managed private hosted cluster on {aws-first} fails because the `bootstrap-kubeconfig` file uses an incorrect `kube-apiserver` port. As a result, the {aws-short} instances are provisioned but cannot join the hosted cluster as nodes. With this release, the issue is fixed so this issue no longer occurs. (link:https://issues.redhat.com/browse/OCPBUGS-42214[*OCPBUGS-42214*])
```